node/primitives/bridge/src/lib.rs

```diff
@@ -180,6 +190,16 @@ impl Chain for Pangolin {
 	type Balance = Balance;
 	type Index = Nonce;
 	type Signature = Signature;
+
+	// TODO: S2S
+	fn max_extrinsic_size() -> u32 {
+		todo!()
+	}
+
+	// TODO: S2S
+	fn max_extrinsic_weight() -> Weight {
+		todo!()
+	}
```

Comment:
what's the function of these two methods. Do we need to complete these todo right now or later?